client/my-sites/hosting/sftp-card/index.js

```diff
@@ -189,6 +265,25 @@ export const SftpCard = ( {
 							}
 						) }
 					</PanelBody>
+					{ siteHasSshFeature && (
+						<PanelBody title={ translate( 'What is SSH?' ) } initialOpen={ false }>
+							{ translate(
+								'SSH stands for Secure Shell. It’s a way to perform advanced operations on your site using the command line. ' +
+									'For more information see {{supportLink}}SFTP on WordPress.com{{/supportLink}}.',
+								{
+									components: {
+										supportLink: (
+											<ExternalLink
+												icon
+												target="_blank"
+												href={ localizeUrl( 'https://wordpress.com/support/sftp/' ) }
```

Comment:
Should we update the support link? or at least create an issue to do it in the near future?